### PR TITLE
add bytecode circuit into aggregator circuit

### PR DESCRIPTION
**Pull request by DreamWuGit**

### CODE REVIEW

File: zkevm/tests/aggregation_tests.rs [hunk 1]
```diff
@@ -97,9 +96,9 @@ fn verifier_circuit_verify(d: &str) {
     assert!(verifier.verify_agg_circuit_proof(agg_proof).is_ok())
 }
 
-#[cfg(feature = "prove_verify")]
+//#[cfg(feature = "prove_verify")]
 #[test]
-fn test_4in1() {
+fn test_5in1() {
```
Comment (lispc) [inline zkevm/tests/aggregation_tests.rs:101]: change it to test_agg ... so we don't need to change it to test_6in1 later ..

Comment (DreamWuGit) [reply zkevm/tests/aggregation_tests.rs:101]: good to me.

Comment (DreamWuGit) [reply zkevm/tests/aggregation_tests.rs]: updated in [735d170](https://github.com/scroll-tech/common-rs/pull/55/commits/735d170eb1488ec480fecaa25fa3e34093a228a0)

**Comment by DreamWuGit** [comment]
close this now as forward in #61

**Pull request closed without merging**